ide/dlight.nativeexecution/tools/Makefile

```diff
@@ -32,8 +32,8 @@ CF_Linux-x86_64	    = -DLINUX   -D_GNU_SOURCE -D_XOPEN_SOURCE=700 -m64
 CF_Linux-sparc_64   = -DLINUX   -D_GNU_SOURCE -D_XOPEN_SOURCE=700 -m64
 CF_Linux-arm	    = -DLINUX   -D_GNU_SOURCE -D_XOPEN_SOURCE=700 -std=c99
 CF_Linux-aarch64    = -DLINUX   -D_GNU_SOURCE -D_XOPEN_SOURCE=700 -std=c99
-CF_MacOSX-x86	    = -DMAXOSX  -D_DARWIN_C_SOURCE -m32
-CF_MacOSX-x86_64    = -DMAXOSX  -D_DARWIN_C_SOURCE -m64
+CF_MacOSX-arm	    = -DMAXOSX  -D_DARWIN_C_SOURCE -target arm64-apple-macos
```

Comment:
We're changing these Makefiles by hand, and that won't allow opening them again using NetBeans CND. 
That's something we may want to tackle in the future, by adding the new platform "arm64-apple-macos" to CND.

Reply:
Ah, ok, something to look at later. This change is based on @oyarzun suggestion. It works to compile until we get Apple Silicon runners, which should be soon.

Ideally I think we would move to having a single, universal binaries for these with macOS anyway?

Reply:
https://github.blog/changelog/2023-10-02-github-actions-apple-silicon-m1-macos-runners-are-now-available-in-public-beta/ they are now available btw

Reply:
I only checked yesterday! 🤣